docs/notebooks/jax-101/01-jax-basics.md

```diff
@@ -0,0 +1,375 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.0
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "6_117sy0CGEU"}
+
+# JAX as accelerated Numpy
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jax/blob/master/docs/notebooks/jax-101/01-jax-basics.ipynb)
+
+In this first colab you will learn the very fundamentals of JAX.
+
++++ {"id": "CXjHL4L6ku3-"}
+
+## Getting started with JAX numpy
+
+Fundamentally, JAX is a library that enables transformations of array-manipulating programs written with a Numpy-like API. 
+
+Over the course of this series of guides, we will unpack exactly what that means. For now, you can think of JAX as *differentiable Numpy that runs on accelerators*.
+
+The code below shows how to import JAX and create a vector.
+
+```{code-cell} ipython3
+:id: ZqUzvqF1B1TO
+
+import jax
+import jax.numpy as jnp
+
+x = jnp.arange(10)
+print(x)
+```
+
++++ {"id": "rPBmlAxXlBAy"}
+
+So far, everything is just like Numpy. A big appeal of JAX is that you don't need to learn a new API. Many common Numpy programs would run just as well in JAX if you substitute `np` for `jnp`. However, there are some important incompatibilities which we touch on at the end of this colab.
```

Comment:
```suggestion
So far, everything is just like NumPy. A big appeal of JAX is that you don't need to learn a new API. Many common NumPy programs would run just as well in JAX if you substitute `np` for `jnp`. However, there are some important incompatibilities which we touch on at the end of this Colab.
```

Would "differences" be also OK in place of "incompatibilities"? 🤷‍♂️  Or maybe I'm overthinking